tests/unit_test/app/routes/git_tokens.py

```
@@ -0,0 +1,463 @@
+			
+			mock_git_label.filter.assert_called_once_with(
+				user_id="user_abc123", label=label
+			)
+
+
+class TestAddGitTokenEndpoint:
+	"""Test cases for POST /api/v1/git_tokens/ endpoint."""
+	
+	@staticmethod
+	def get_url() -> str:
+		return f"/api/v1/git_tokens/"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)